### PR TITLE
fix (docs): indicate that custom domains for postgres doesn't require configuration

### PR DESCRIPTION
**Pull request by dbarrosop**

### CODE REVIEW

File: docs/platform/custom-domains.mdx [hunk 1]
```diff
@@ -34,14 +34,13 @@ fqdn = ['hasura.custom-domain.com']
...
 
-[[postgres.resources.networking.ingresses]]
-fqdn = ['postgres.custom-domain.com']
-
 [[functions.resources.networking.ingresses]]
 fqdn = ['functions.custom-domain.com']
 ```
 
-For Run services, typically in nhost-service.toml specific to the service:
+For PostgreSQL there is nothing to configure on Nhost's side so as long as you have a CNAME properly configured it should work.
```
Comment (quentin-decre) [inline docs/platform/custom-domains.mdx:41]: I do not understand this. How do you know that postgres.develop.jarvi.tech should point to postgres service ?

File: docs/platform/custom-domains.mdx [hunk 2]
```diff
@@ -34,14 +34,13 @@ fqdn = ['hasura.custom-domain.com']
...
+
+For Run services, typically in an `nhost-service.toml` specific to the service:
```
Comment (quentin-decre) [inline docs/platform/custom-domains.mdx:44]: Also maybe, in the doc, a tab or link redirecting to how to setup custom domains in the the sdk would be useful

File: docs/platform/custom-domains.mdx [hunk 3]
```diff
@@ -34,14 +34,13 @@ fqdn = ['hasura.custom-domain.com']
 [[auth.resources.networking.ingresses]]
 fqdn = ['auth.custom-domain.com']
 
-[[postgres.resources.networking.ingresses]]
-fqdn = ['postgres.custom-domain.com']
-
 [[functions.resources.networking.ingresses]]
 fqdn = ['functions.custom-domain.com']
 ```
```
Comment (quentin-decre) [inline docs/platform/custom-domains.mdx:39]: What about storage ?

File: docs/platform/custom-domains.mdx (hunk 1 above)
Comment (dbarrosop) [reply docs/platform/custom-domains.mdx:41]: We don't care. As long as you connect to the right IP (which will depend on your correct CNAME configuration) and you have the correct credentials and database you are good. With the rest of the services we need to know your domain or choice to add it to the load balancer and to generate TLS certificates but here that's not needed.

**Review by nunopato — APPROVED** [review]

**Pull request merged**